src/normal_form_game.jl

```diff
@@ -691,3 +691,112 @@ Return true if `action_profile` is Pareto dominant for game `g`.
 
 * `::Bool`
 """ is_pareto_dominant
+
+# is_dominated
+
+"""
+    is_dominated(player, action[, tol=1e-8, lp_solver=ClpSolver()])
+
+Determine whether `action` is strictly dominated by some mixed
+action.
+
+# Arguments
+
+- `player::Player` : Player instance.
+- `action::PureAction` : Integer representing a pure action.
+- `tol::Float64` : Tolerance to be used.
+- `lp_solver::AbstractMathProgSolver` : Allows users to choose a particular
+  solver for linear programming problems. Options include ClpSolver(),
+  CbcSolver(), GLPKSolverLP() and GurobiSolver(). By default, it choooses
+  ClpSolver().
+
+# Returns
+
+- `::Bool` : True if `action` is strictly dominated by some mixed action;
+  False otherwise.
+
+"""
+function is_dominated(player::Player, action::PureAction;
+                      tol::Float64=1e-8,
+                      lp_solver::MathProgBase.AbstractMathProgSolver=
+                      ClpSolver())
+    payoff_array = player.payoff_array
+    if eltype(player.payoff_array) == Rational
+        input_game_type = Rational
+    else
+        input_game_type = Float64
+    end
```

Comment:
Follow [this approach](https://github.com/QuantEcon/Games.jl/blob/34c99fc875cf97eccf965146217c9d796560110b/src/support_enumeration.jl#L99) in `support_enumeration`. Note that `Player` has a type parameter `T`.